app/assets/javascripts/AngularConfig.js.coffee

```diff
@@ -610,18 +610,29 @@ angular.module('dahlia.controllers',['ngSanitize', 'angular-carousel', 'ngFileUp
     AccountService.validateUser()
 
     $rootScope.$on '$stateChangeStart', (e, toState, toParams, fromState, fromParams) ->
+
+      # change logged in state of user
       if (ShortFormApplicationService.isLeavingShortForm(toState, fromState))
-          # timeout from inactivity means that we don't need to ALSO ask for confirmation
-          if (toParams.skipConfirm || $window.confirm($translate.instant('T.ARE_YOU_SURE_YOU_WANT_TO_LEAVE')))
-            # disable the onbeforeunload so that you are no longer bothered if you
-            # try to reload the listings page, for example
-            $window.removeEventListener 'beforeunload', ShortFormApplicationService.onExit
-            ShortFormApplicationService.resetUserData()
-            AccountService.rememberShortFormState(null)
-          else
-            # prevent page transition if user did not confirm
-            e.preventDefault()
-            false
+        # Boolean for Logged in Users on the confirmation page of short form to remove the leave confirmation.
+        loggedInConfirmation = (AccountService.loggedIn() && fromState.name == 'dahlia.short-form-application.confirmation') ? true : false
```

Comment:
@menslow 
the `#change logged in state of user` comment seems out of place, can remove?

also can remove the `? true : false` since it's already returning a boolean.